code/datums/elements/organ_set_bonus.dm

```diff
@@ -80,10 +84,21 @@
 		if((required_biotype == MOB_ORGANIC) && !owner.can_mutate())
 			return FALSE
 	bonus_active = TRUE
+	// Add traits
 	if(length(bonus_traits))
 		owner.add_traits(bonus_traits, REF(src))
+
+	// Add biotype
+	if(owner.mob_biotypes & bonus_biotype)
+		biotype_added = FALSE
+		return TRUE
+	owner.mob_biotypes |= bonus_biotype
+	biotype_added = TRUE
```

Comment:
So if the mob already has the biotype we don't add anything else? No limb overlays, no activation text? Why?

Reply:
whoops